code/game/machinery/teleporter.dm

```diff
@@ -168,6 +168,8 @@
 /obj/machinery/computer/teleporter/proc/set_target(var/obj/O)
 	src.locked = O
 	GLOB.destroyed_event.register(locked, src, .proc/target_lost)
```

Comment:
` .proc/target_lost` doesn't appear to reset the portal visuals, which would result in a GC error due to the portal holding a reference to the destination atom